client/settings/deposits/index.js

```diff
@@ -235,7 +235,7 @@ const Deposits = () => {
 					) : (
 						<p className="deposits__bank-information-help">
 							{ __(
-								'Manage and update your deposit account information to receive payments and deposits.',
+								'Manage and update your payout account information to receive payments and payouts.',
```

Comment:
This might need tweaking, second payouts seems redundant. 

Would this communicate everything we need to say here?

> Manage and update your payout account information to receive payments.

Reply:
Or maybe:

> Manage and update your bank account information to receive payouts.

Reply:
I went ahead with

> Manage and update your bank account information to receive payouts.

[30ac371](https://github.com/Automattic/woocommerce-payments/pull/9532/commits/30ac37106c2d8723d58e2f62e6a4367fbbda733f)